organize/settings.mdx

```diff
@@ -134,7 +134,10 @@ This section contains the full reference for the `docs.json` file.
       The style of the page eyebrow. Choose `section` to show the section name or `breadcrumbs` to show the full navigation path. Defaults to `section`.
     </ResponseField>
     <ResponseField name="latex" type="boolean">
-      Controls whether LaTeX stylesheets are included, overriding automatic detection.
+      Controls whether LaTeX stylesheets are included, overriding automatic detection. By default, Mintlify automatically detects LaTeX usage in your content and loads the necessary stylesheets. Use this setting to:
+      
+      - Set to `true` to force-load LaTeX stylesheets when automatic detection fails to recognize your mathematical expressions
+      - Set to `false` to prevent loading LaTeX stylesheets for improved performance if you don't use mathematical expressions but have content that triggers false-positive detection
```

Comment:
```suggestion
      Controls whether LaTeX stylesheets are included, overriding automatic detection. By default, Mintlify automatically detects LaTeX usage in your content and loads the necessary stylesheets.
      
      - Set to `true` to force-load LaTeX stylesheets when automatic detection fails to recognize your mathematical expressions.
      - Set to `false` to prevent loading LaTeX stylesheets for improved performance if you don't use mathematical expressions but have content that triggers false-positive detection.
```